src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/bdd/BddSteps.kt

```diff
@@ -101,6 +101,70 @@ class BddSteps : En {
     Given("an offender is {string}") { gender: String ->
       setupData.setGender(gender)
     }
+
+    Given("an offender scores 21 points") {
+      setupData.setValidAssessment()
```

Comment:
maybe we could set this to be valid within setNeeds or somewhere else appropriate?